src/App.css

```diff
@@ -1,8 +1,12 @@
+:root {
+    --blue: #3C465F;
```

Comment:
why is this actually used for ?

Reply:
To define colors globally, so that I can use it in any of the stylesheets.

Reply:
CSS Custom Properties (Variables), launched in 2020 to define themes globally.